### PR TITLE
Migrate static HTML pages to JSP-generated #5755

### LINKED ISSUE #5755: Migrate static HTML pages to JSP-generated
The static pages such as `about.html`, `index.html`, `contact.html`, ... all have the exact same header/footer and keeping them as `.html` necessitates repetition (and thus variations) of such. It will be painful for whoever is going to do #3362.

Also, when migrated to JSP, we can harness the power of tags, e.g using tag to construct about page's name/picture list etc. Not to mention that we can also pull constants from `Const.java` and properties from `build.properties`. (case in point: teammates@comp.nus.edu.sg still hardcoded in many places) (case in point 2: look at that dreadfully long `instructorHelp.html`; it can be constructed with existing tags)

As long as no data from back end is pulled (which is the case), this migration would not have any visible drawback.

Add: this excludes HTML templates (question forms, emails) because their functions are different.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #5755

I haven't done tag extractions within each page yet (e.g the contributors list on the about page, the overview bytes of the index page, etc); only the headers and footers.

Since the changes all happen on static pages, the best way to confirm is by testing the pages directly: have a look at [my staging server](http://5-81sp-dot-teammates-wkurniawan07.appspot.com) (there's nothing to see though, other than that all the static pages are still working). In particular, review the `instructorHelp.jsp` page since the content is too big to be displayed in the GitHub diff view.

Anyone doing #3362 can simply modify `staticPage.tag` and all affected pages will be modified accordingly; no need to duplicate the change 7 times.

### CODE REVIEW

**Comment by chowyb** [comment]
Behaviour of the upper navbar for `instructorHelp` is now different, under the HTML version it has a `relative` position, while in the JSP form it has a `fixed` position, which isn't too big of a problem, but when clicking hyperlinks to other parts of the page, the relevant heading/question tends to be hidden under the navbar. Might be good to change that

**Comment by wkurniawan07** [comment]
I see what you mean. Should be able to fix that no sweat. Anything else to add?

**Comment by chowyb** [comment]
Hmm nothing else about behaviour, looking through for style now but I doubt I'll find anything haha

**Comment by chowyb** [comment]
`instructorHelp.jsp`, line 2130:

`<span >` should be `<span>`, unless @YongJieYongJie wants to address stuff like this in another issue =P

(will be editing this comment if anything else is found in `instructorHelp.jsp`)

**Comment by wkurniawan07** [comment]
@chowyb spare me on that, I have no intention of tidying up that dreadful file... 😨 (because like I said in the issue thread it'd be so much better if it's built with existing tags)
Anyway, the upper navbar in the staging server has been updated as well, you can take a look.

**Comment by chowyb** [comment]
Alright navbar looks good.

I'm assuming fixes in the other files are fair game then =P

**Comment by wkurniawan07** [comment]
@chowyb I did tidy up the other static pages though, all of them are < 200 lines and straightforward, except the about page which has more (due to the large number of contributors) but it's not too hard to tidy up as well.

File: src/main/webapp/studentHelp.jsp [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        <li>
+            <div>
+                <b>What should I do if I cannot see any/some of my courses I should be able to see in the system?</b>
+            </div>
+            <div>
+                These are the possible reasons:
+                <ol>
+                    <li>
+                        You used a different Google account to access TEAMMATES in the past.
+                        In that case, you need to  use the same Google account to access TEAMMATES again.
```
Comment (chowyb) [inline src/main/webapp/studentHelp.jsp]: Double whitespace here

File: src/main/webapp/studentHelp.jsp [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+                        Logout and re-login using the other Google account.</li>
```
Comment (chowyb) [inline src/main/webapp/studentHelp.jsp]: Maybe shift the `</li>` to the next line to be consistent with the next list item

**Comment by chowyb** [comment]
Alright good to go :+1:

**Pull request merged**